docs/pages/build-reference/android-builds.md

```diff
@@ -42,12 +42,16 @@ Next, this is what happens when EAS Build picks up your request:
 
    - `COMMAND` is the command defined in your **eas.json** at `builds.android.PROFILE_NAME.gradleCommand`. It defaults to `:app:bundleRelease` which produces the AAB (Android App Bundle).
 
-1. Run the `eas-build-pre-upload-artifacts` script from package.json if defined.
+1. **Deprecated:** Run the `eas-build-pre-upload-artifacts` script from package.json if defined.
```

Comment:
```suggestion
1. **Deprecated:** Run the `eas-build-pre-upload-artifacts` script from **package.json** if defined.
```

I know this unrelated to this PR, but I think [bolding the file names](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text) would allow you us to keep the writing style consistent through the docs 👍 Just adding this as a suggestion.